chia/types/blockchain_format/proof_of_space.py

```diff
@@ -57,15 +62,13 @@ def verify_and_get_quality_string(
         return None
     plot_id: bytes32 = get_plot_id(pos)
     new_challenge: bytes32 = calculate_pos_challenge(plot_id, original_challenge_hash, signage_point)
-
     if new_challenge != pos.challenge:
```

Comment:
```suggestion

    if new_challenge != pos.challenge:
```

nit, feel free to just close this out but i still want to point to it because its unrelated here.

Reply:
Thanks, taken.